include/lm/diff.h

```diff
@@ -0,0 +1,166 @@
+/**
+ * @file diff.h
+ * @author Sean Massung
+ *
+ * All files in META are dual-licensed under the MIT and NCSA licenses. For more
+ * details, consult the file LICENSE.mit and LICENSE.ncsa in the root of the
+ * project.
+ */
+
+#ifndef META_LM_DIFF_H_
+#define META_LM_DIFF_H_
+
+#include <unordered_map>
+#include <unordered_set>
+#include "cpptoml.h"
+#include "lm/language_model.h"
+
+namespace meta
+{
+namespace lm
+{
+class diff
+{
+  public:
+    /**
+     * @param config_file The file containing configuration information
+     */
```

Comment:
What is the configuration information? Maybe an example toml snippet would be useful.